src/com/company/Main.java

```diff
@@ -17,8 +17,7 @@
     private static final String WIN_OUTPUT_FORMAT = "%s %s %d\n";
 
     private static final Scanner INPUT = new Scanner(System.in);
-    private static int boardSize = 0;
-    private static boolean shouldPlayAgain;
+    private static boolean PlayAgain;
```

Comment:
dlaczego zmienna ma nazwę z dużej litery? proszę popraw.
Pozatym skąd ta zmiana? mówiłem ci, że zmienne typu boolean powinny mieć przedrostki takie jak: `is`, `has`, `should` itp. tak aby zadawały pytanie na które odpowiedzią jest przechowywana przez nie wartość.

Reply:
zakręciłam się, wróciłam do przedrostka should